seedlet/assets/sass/components/comments/_comment-form.scss

```diff
@@ -23,6 +23,11 @@
 	.url {
 		border-bottom: none;
```

Comment:
I wonder if we should just remove this line, so the behaviour is the same as on the rest of the theme, i.e. the link is underlined until it's hovered.